### ISSUE TITLE
Improving navigation with thumb buttons

### ISSUE THREAD

**Issue by akaKarthick** [czbar/ChessForge#680]
I'm writing to ask if you could make it possible to assign the thumb buttons to the "Go back" and "Go forward" actions. Currently, there is no keyboard shortcut for these actions.

![Screenshot (393)](https://github.com/czbar/ChessForge/assets/125844120/cff4da03-3827-4438-a881-d73e933cd7d0)
I think this would be a really helpful addition because it would make it easier to navigate workbook studies. 

What's your thought?

**Comment by czbar** [comment]
Yes, good suggestion. I am thinking of assigning ALT + LEFT ARROW and ALT + RIGHT ARROW because these are the "back/forth" shortcuts in the Chrome/Edge browsers.

**Comment by akaKarthick** [comment]
So both will be added right?![Screenshot_20230823-011806_Google.png](https://github.com/czbar/ChessForge/assets/125844120/15ded061-ddb2-434e-af94-1b3dff33834f)

**Comment by czbar** [comment]
Not these two :-) but yes.

**Comment by czbar** [comment]
Implemented in private 1.2.6.  Note that the navigation itself is not perfect yet (e.g. issue  #537).
